extensions/wikia/CoppaTool/RenameIPLogFormatter.class.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/**
+ * @author: Władysław Bodzek
+ *
+ * A helper class for the User rename tool
+ * @copyright (C) 2010, Wikia Inc.
+ * @licence GNU General Public Licence 3.0 or later
+ */
+class RenameIPLogFormatter {
+	static public function getCommunityUser( $name, $noRedirect = false ) {
+		if ( is_int( $name ) )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add curly braces around the nested statement(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)

[project-id]: wikia:app